client/src/App.js

```diff
@@ -1,12 +1,47 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from "react";
+import { Link, Route, Switch, BrowserRouter as Router } from "react-router-dom";
+import "./App.css";
+import ChatOne from "./chat1.js";
+import ChatTwo from "./chat2.js";
```

Comment:
These variables should be a little more specific so that you know which chat room you're looking at right away if you were to come back and work on this more, or for other people who are looking at it